### PR TITLE
Updated Spectator UI layout

### PR DESCRIPTION
**Pull request by Flipybitz**
Spectating online ghost:
Screenshot_1
Spectating a replay:
Screenshot_2

Basically just changed the font for the username, centered it along with the time, and laid it out nicely :)

### CODE REVIEW

File: mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp [hunk 1]
```diff
@@ -62,7 +62,7 @@ CMOMSpectatorGUI::CMOMSpectatorGUI(IViewPort *pViewPort) : EditablePanel(nullptr
     SetScheme("ClientScheme");
 
     m_pTopBar = new Panel(this, "TopBar");
-    m_pPlayerLabel = new Label(this, "PlayerLabel", "#MOM_ReplayPlayer");
+    m_pPlayerLabel = new Label(this, "PlayerLabel", "");//set below on line 283
```
Comment (SlidyBat) [inline mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp]: Don't make comments that refer to line numbers, that'll quickly become stale as soon as someone adds/removes a line in this file and it'll just confuse future readers. I don't think this comment is needed at all, but if you really want it you can change it to something like "set in Update()"

Comment (Flipybitz) [reply mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp]: Good point, my bad

File: mp/game/momentum/resource/ui/Spectator.res [hunk 2]
```diff
@@ -32,60 +32,61 @@
...
+        "tall"				"22"
         "autoResize"		"0"
         "pinCorner"			"0"
         "visible"			"1"
         "enabled"			"1"
-        "textAlignment"		"west"
         "dulltext"			"0"
         "brighttext"		"0"
-        "auto_wide_tocontents" "1"
+		"font"				"BigNoodleSmall"
```
Comment (Gocnak) [inline mp/game/momentum/resource/ui/Spectator.res]: Convert the spacing of this file to spaces (in a new commit) to be consistent with the file.

File: mp/game/momentum/resource/momentum_english_ref.res [hunk 3]
```diff
@@ -372,7 +372,6 @@
         // Spec GUI
         "MOM_WatchingReplay"    "REPLAY"
         "MOM_WatchingGhost"    "SPECTATING"
-        "MOM_ReplayPlayer"      "Player: %s1"
```
Comment (Gocnak) [inline mp/game/momentum/resource/momentum_english_ref.res]: Something to be careful of is making sure this token isn't used anywhere else. While this a spectator GUI specific string, some strings are shared among panels. Make sure to search other .res files for tokens if you ever remove them.

File: mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp (hunk 1 above)
Comment (Gocnak) [reply mp/src/game/client/momentum/ui/spectate/mom_spectator_gui.cpp]: You can remove the comment. Anyone reading the code will very clearly see where it's set; this comment is unnecessary.

File: mp/game/momentum/resource/momentum_english_ref.res (hunk 3 above)
Comment (Flipybitz) [reply mp/game/momentum/resource/momentum_english_ref.res]: I don't believe it's used anywhere else thankfully, but next time I will definitely be more careful. Thanks for the advice :)

**Comment by Gocnak** [comment]
Rebase onto latest develop and rerun tokenizer before you force push.

**Comment by Gocnak** [comment]
If you can rebase onto latest develop and fix the conflicts (rerun tokenizer) this should be good to go.

**Pull request merged**